test/integration/spec/localparticipant.js

```diff
@@ -29,7 +30,12 @@ const defaults = require('../../lib/defaults');
 const { isChrome, isFirefox, isSafari } = require('../../lib/guessbrowser');
 const { createRoom, completeRoom } = require('../../lib/rest');
 const getToken = require('../../lib/token');
-const { isRTCRtpSenderParamsSupported, smallVideoConstraints } = require('../../lib/util');
+
+const {
+  createSyntheticAudioStreamTrack,
+  isRTCRtpSenderParamsSupported,
+  smallVideoConstraints
+} = require('../../lib/util');
```

Comment:
you can combine these with the next line, which also imports from '../../lib/util'